tools/cockroachdb_version

```diff
@@ -1 +1 @@
-v22.1.9
+v22.1.22-25-g24e7b94071
```

Comment:
out of curiosity, what's this new version string mean?

Reply:
This is a [git-describe](https://git-scm.com/docs/git-describe) string, read as "25 commits after the tag `v22.1.22`, at commit `24e7b94071`".